src/tools/gdbjit.c

```diff
@@ -34,15 +34,15 @@ typedef struct gdbjit_descriptor_s {
 } gdbjit_descriptor_t;
 
 /* GDB puts a breakpoint in this function. This can't be optimized out. */
-void __attribute__((noinline)) __jit_debug_register_code()
+EXPORT void __attribute__((noinline)) __jit_debug_register_code()
```

Comment:
the noinline attribute is probably no longer needed with the `EXPORT` attribute.